js/main.js

```diff
@@ -462,13 +462,18 @@ function fixMapHeight() {
 $(window).on('resize', fixMapHeight);
 
 
-$(window).on('hashchange',function() {
-    // Don't create a new history state, because the hash change already did
-    setCity(getPathCity(), false);
-});
-
+// we used to pass the city after the `#` in the url.
+// now we simply put it after the `/`
+// so /#berlin becomes /berlin
+function removeHashPrefixInUrl(){
```

Comment:
Sorry, I found a another missing space here. We should add automatic formatting anytime soon.